src/test/cucumber/per_project_configuration.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Per-project configuration
+
+  Background:
+    Given the internet is reachable
+    And an initialised environment
+
+  Scenario: Env without providing an .sdkmanrc file
```

Comment:
_(the one where)_ The env command is issued without an sdkman project configuration present

I like framing these scenario headlines in the way that Friends episodes are titled :smile: